triaging.rst

```diff
@@ -47,6 +47,35 @@ the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
    Existing members with the "Developer" role on the `issue tracker`_ are welcome to
    self-nominate for triage team membership if they are interested.
+   
+GitHub Labels
+''''''''''''''''
+
+An important component of triaging PRs and issues involves appropriately 
+categorizing them through the usage of labels. Here are several of the
+common ones utilized by the Python triage team:
+
++-----------------------+----------------------------------------------------------------------------+
+| Label                 | Description                                                                |
++-----------------------+----------------------------------------------------------------------------+
+| needs backport to X.Y | Used for PRs which are appropriate to backport to branches prior to        |
+|                       | master. Generally, backports to the maintenance branches are primarily     |
+|                       | bugfixes and documentation clarifications. Backports to the security       |
+|                       | branches are strictly reserved for PRs involving security fixes, such as   |
+|                       | crashes, privilege escalation, and DoS.                                    |
++-----------------------+----------------------------------------------------------------------------+
+| skip issue            | Used for PRs which involve trivial changes, such as typo fixes,            |
+|                       | comment changes, and section rephrases. The majority of PRs require        |
+|                       | an issue to be attached to, but if there are no code changes and the       |
+|                       | section being modified retains the same meaning, this label might be       |
+|                       | appropriate. It is also used commonly for backports.                       |
++-----------------------+----------------------------------------------------------------------------+
+| skip news             | Used frequently in combination with the skip issue label for PRs that      |
```

Comment:
I don't think we need to mention "used frequently in combination with ... ".
The reason for applying this label is more about the triviality of the PR, not whether the skip issue was added or not.

Reply:
The main reason I used that was to avoid redundantly defining how a PR could be considered trivial, since the skip news and skip issue labels are both used frequently on PRs involving trivial changes. Perhaps I should instead phrase it like "Similar to skip issue..." instead of making it sound dependent upon skip issue.